src/cloudai/workloads/ucc_test/slurm_command_gen_strategy.py

```diff
@@ -39,6 +39,12 @@ def generate_test_command(self) -> List[str]:
         srun_command_parts.append(f"-c {tdef_cmd_args.collective}")
         srun_command_parts.append(f"-b {tdef_cmd_args.b}")
         srun_command_parts.append(f"-e {tdef_cmd_args.e}")
+        if tdef_cmd_args.gen is not None:
```

Comment:
List in CmdArgs classes are for sweeps, they are unrolled during the run.

Inside `generate_test_command()` we assume that list for an arg is not possible, so let's do simple `srun_command_parts.append(f"--gen {tdef_cmd_args.gen}")` only when `gen` is set.

Unless setting them all in a single cmd is what you need, of course. But in this case we need another way to approach, please let me know if this is the case.